$NetBSD$

Repair klauncher support for unique-applications like konsole.
  (adding org.kde.konsole in konsole.desktop was just a workaround, org.kde
   is still assumed when not present)

svn 918654

--- branches/KDE/4.2/kdelibs/kinit/klauncher.h	2009/01/30 12:45:23	918653
+++ kinit/klauncher.h	2009/01/30 12:46:15	918654
@@ -87,6 +87,7 @@
    QString name;
    QStringList arg_list;
    QString dbus_name;
+   QString tolerant_dbus_name;
    enum status_t { Init = 0, Launching, Running, Error, Done };
    pid_t pid;
    status_t status;
